canonical-websites/www.ubuntu.com
# Added canonical-global-nav as a dependency

## Done
Added canonical-global-nav as a dependency and removed all the old global navigation code.

## QA
- Pull branch and run `./run`
- Go to http://0.0.0.0:8001/
- See that the global navigation is there and works
- Check small screens

Demo: http://www.ubuntu.com-global-nav-component.demo.haus
